hs/src/Reach/Linearize.hs

```diff
@@ -603,14 +603,15 @@ df_step :: DKTail -> DFApp LLStep
 df_step = \case
   DK_Stop at -> return $ LLS_Stop at
   DK_ToConsensus at send recv mtime -> do
-    lt <- fmap snd <$> fluidRefm FV_thisConsensusTime
+    lt <- fmap snd <$> fluidRefm FV_lastConsensusTime
```

Comment:
Was this just wrong before? I'm not sure I grasp how both the previous code and the current code for this line could both be okay, since these concepts are not interchangeable.

Reply:
Oh ya, good catch. It was CORRECT before, because the way we learn what time it was is looking at what the old "this" time WAS